### PR TITLE
Agent release for ciprod05192022 and win-ciprod05192022 

### PR DESCRIPTION
**Pull request by jatakiajanvi12**
- Updates for agent release ciprod05192022 and win-ciprod05192022 

1. Release notes 
2. Dockerfile
3. Yaml
4. Chart

### CODE REVIEW

**Review by ganga1980 — APPROVED** [review]

File: ReleaseNotes.md [hunk 1]
```diff
@@ -11,6 +11,43 @@ additional questions or comments.
...
+- Linux Agent
+  - PodReadyPercentage metric bug fix
+  - add cifs & fuse file systems to ignore list
+  - CA Cert Fix for Mariner Hosts in Air Gap
+- Windows Agent
+  - Ruby version upgrade from 2.6.5.1 to 2.7.5.1
+  - Added Support for Windows Server 2022
+  - Multi-Arch Image to support both Windows 2019 and Windows 2022
+- Common (Linux & Windows Agent)
+  - Telegraf version update from 1.20.3 to 1.22.2 to fix the vulnerabilitis
```
Comment (gracewehner) [inline ReleaseNotes.md:27]: Could we please add a note that disk usage metrics will no longer be collected for the paths "/mnt/containers" and "/mnt/docker" ?

File: ReleaseNotes.md [hunk 2]
```diff
@@ -11,6 +11,43 @@ additional questions or comments.
...
+##### Code change log
+- Linux Agent
+  - PodReadyPercentage metric bug fix
+  - add cifs & fuse file systems to ignore list
+  - CA Cert Fix for Mariner Hosts in Air Gap
+- Windows Agent
+  - Ruby version upgrade from 2.6.5.1 to 2.7.5.1
+  - Added Support for Windows Server 2022
+  - Multi-Arch Image to support both Windows 2019 and Windows 2022
+- Common (Linux & Windows Agent)
```
Comment (ganga1980) [inline ReleaseNotes.md:26]: @jatakiajanvi12 , we also need to add this line under common "configmap updates for log collection settings for v2 schema"

Comment (jatakiajanvi12) [reply ReleaseNotes.md:27]: Updated!

File: ReleaseNotes.md (hunk 1 above)
Comment (jatakiajanvi12) [reply ReleaseNotes.md:28]: Updated!

**Review by ganga1980 — APPROVED** [review]

**Pull request merged**